assets/js/base/components/cart-checkout/order-summary/style.scss

```diff
@@ -69,7 +73,6 @@
 
 .wc-block-components-order-summary-item__description {
 	padding-left: $gap-large;
-	padding-top: $gap;
 	padding-bottom: $gap;
 	line-height: 1.375;
```

Comment:
I wonder if we should remove this `line-height` rule. Currently the product name and price are not vertically aligned because of it:
![imatge](https://user-images.githubusercontent.com/3616980/105848829-d755c400-5fdf-11eb-83c8-7397a11d48aa.png)

Reply:
Yep, looks nicer without 👍🏻